arangod/Pregel/Conductor/WorkerApi.cpp

```diff
@@ -34,30 +40,29 @@ auto WorkerApi::execute(In const& in) const -> futures::Future<ResultT<Out>> {
 
 auto WorkerApi::loadGraph(LoadGraph const& graph)
     -> futures::Future<ResultT<GraphLoaded>> {
-  return execute<GraphLoaded>(graph);
+  return send<GraphLoaded>(*_loadGraphIterator++, graph);
```

Comment:
As mentioned in the PR description, loading the graph is an exception: Here we send different messages to each worker.
I used the iterator here to keep the function signature the same, but it is not optimal: We have to be sure that the given LoadGraph belongs to the ServerID of the iterator.
I had the idea we change the LoadGraph message such that it does not include any DBServer specifics but let the DBServer do the work (find the relevant shards) but this is not save: Due to replication we cannot be sure that we read a shard exactly once - different DBServers could in some use cases assume that they are both the leader for a specific shard and therefore both read the same shard or there is a shard where no DBServer assumes to be the leader of and so it is not read at all.
I'm happy to head suggestions how to handle this case.